core/src/main/java/bio/terra/pearl/core/model/participant/RelationshipType.java

```diff
@@ -1,7 +1,7 @@
 package bio.terra.pearl.core.model.participant;
 
 public enum RelationshipType{
-    PROXY;
+    PROXY, FAMILY;
```

Comment:
There are many places (mostly in the UI) that treat _any_ enrollee relation as providing full access; that is the subject of a future PR.